.github/workflows/linkcheck.yml

```diff
@@ -41,4 +41,15 @@ jobs:
         run: pip install -r requirements_docs.txt
 
       - name: LinkCheck
-        run: make -C doc linkcheck SPHINXOPTS="-W --keep-going --color"
+        run: |
+          make -C doc linkcheck SPHINXOPTS="-W --keep-going --color" > logs.txt || echo "OUTPUT=$(grep -E "broken" logs.txt | tr '\n' '@' | sed 's/@/<br>/g' | sed 's/broken/*\*`broken`\*\*/g' | sed $'s/\e\\[[0-9;:]*[a-zA-Z]//g')" >> $GITHUB_ENV
```

Comment:
Some remarks about this shell command as promised:
1. if I understand correctly, the last bit strips away colour escapes. So why not use `--no-color` instead of `--color` in the `make` call so that we don't have to strip it away later?
2. we don't need to escape the asterisks on the "right" side of `sed 's/broken/*\*`broken`\*\*/g'` (in fact one escape is missing). In the replacement string `*` is not special so these can just be `**`broken`**`.
3. we can probably join two (or more) consecutive `sed` calls with a single `sed` call with two (or more) `-e` switches, though I haven't tested this one.

Do you perhaps have a screenshot of how this will get rendered? Assuming you tested this somewhere.